### ISSUE TITLE
Using the new reactive pg driver with a limit clause throws a ClassCastException

### ISSUE THREAD

**Issue by schuma7** [jklingsporn/vertx-jooq#61]
I'm currently experimenting with the new reactive PostgreSQL driver: io.github.jklingsporn:vertx-jooq-classic-reactive:4.0.0-BETA.

Executing the following query without a limit clause is working:

`queryExecutor.execute(dslContext -> dslContext.selectFrom(EXCHANGE))`

While executing the following statement throws an exception:

`queryExecutor.execute(dslContext -> dslContext.selectFrom(EXCHANGE).limit(2))`

```
SEVERE: Unhandled exception
java.lang.ClassCastException: java.lang.Integer cannot be cast to java.lang.Long
	at io.reactiverse.pgclient.impl.codec.DataType$7.encodeBinaryInternal(DataType.java:222)
	at io.reactiverse.pgclient.impl.codec.DataType.encodeBinary(DataType.java:1059)
	at io.reactiverse.pgclient.impl.codec.encoder.message.Bind.encode(Bind.java:154)
	at io.reactiverse.pgclient.impl.codec.encoder.message.Bind.encode(Bind.java:167)
	at io.reactiverse.pgclient.impl.SocketConnection.checkPending(SocketConnection.java:210)
	at io.reactiverse.pgclient.impl.SocketConnection.bilto(SocketConnection.java:191)
	at io.reactiverse.pgclient.impl.CommandBase.foo(CommandBase.java:54)
	at io.reactiverse.pgclient.impl.SocketConnection.schedule(SocketConnection.java:181)
	at io.reactiverse.pgclient.impl.ConnectionPool$PooledConnection.schedule(ConnectionPool.java:81)
	at io.reactiverse.pgclient.impl.PgPoolImpl$1.lambda$onSuccess$0(PgPoolImpl.java:73)
	at io.vertx.core.impl.ContextImpl.lambda$wrapTask$2(ContextImpl.java:339)
	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:163)
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:404)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:463)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:886)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.lang.Thread.run(Thread.java:748)
```

**Comment by jklingsporn** [comment]
Have you tried using `queryExecutor.query` instead of `execute`? The `execute` function is used for create, update and delete statements, whereas `query` is used for selects.

**Comment by schuma7** [comment]
I must have copied the wrong line over, when creating the issue. The original code actually used `query`. So that's not the cause of this problem.

**Comment by jklingsporn** [comment]
Hey,
there seems to be an issue when passing integers as a limit into a prepared query in the pg-driver. I've opened https://github.com/reactiverse/reactive-pg-client/issues/188 addressing this.

**Comment by jklingsporn** [comment]
This will be fixed in version 0.10.6 of the PG-driver.
